src/lib/components/NavLink.svelte

```diff
@@ -45,16 +45,4 @@
 			display: inline;
 		}
 	}
-
-	/*Horrible hacky fix on spacing when font is bolded
-	https://stackoverflow.com/questions/556153/inline-elements-shifting-when-made-bold-on-hover */
-	a::after {
-		display: block;
-		content: attr(title);
-		font-weight: bolder;
-		height: 1px;
-		color: transparent;
-		overflow: hidden;
-		visibility: hidden;
-	}
```

Comment:
This one is still needed, at least on Firefox. We use bold text on the currently selected tab/page, and without this there is  slight, but annoying a layout shift on navigation.